package.json

```diff
@@ -484,6 +484,11 @@
           "type": "boolean",
           "default": false,
           "description": "If false, the import statements will be excluded while using the Go to Symbol in File feature"
+        },
+        "go.multiPackage": {
+          "type": "boolean",
```

Comment:
Currently, this flag affects build, lint, vet and test. There might be cases when users might want just a few of these to run on whole workspace. Example: There might be a lot of tests in the workspace and they might take time, so user might not want test to run on whole workspace but only build.

So instead of just having this setting as a simple boolean, we can have it as an object
```
"go.multiPackage": {
          "type": "object",
          "default": {
            "build": false,
            "test": false,
            "lint": false,
            "vet": false
          },
          "description": "Invoke build/lint/test commands recursively from the workspace root."
        }
```

Thoughts?